showcase/src/app/app.module.ts

```diff
@@ -263,12 +266,33 @@ export class AppModule {
 
 		this.settingsService.initializeSettings();
 
-		sessionService
+		this.sessionService
 			.getCurrentUser()
 			.pipe(filter((user?: StarkUser): user is StarkUser => typeof user !== "undefined"))
 			.subscribe((user: StarkUser) => {
 				const devAuthenticationHeaders: Map<string, string> = getAuthenticationHeaders(user);
 				this.sessionService.setDevAuthenticationHeaders(devAuthenticationHeaders);
 			});
+
+		/**
+		 * When the application language changes, the Moment locale and the Material Moment Adapter (internally used by Stark Date Pickers) should be changed as well
+		 */
+		this.sessionService
```

Comment:
It's unfortunately not working when it's part of the **App** module. Could you please move this logic in the **Shared** module to make it work ? 😊